src/traceResolvers.js

```diff
@@ -14,14 +14,16 @@ function fieldPathFromInfo (info) {
 }
 
 const tracer = function (resolver, parent, args, ctx, info) {
+  if (process.env.DISABLE_XRAY_LOGGING) {
```

Comment:
This variable seems misnamed. It seems to disable integration completely, not just logging. 

If you want to disable the resolver tracking, why not skip calling this library completely?